app/views/user_mailer/admin_hidden_series_notification.html.erb

```diff
@@ -0,0 +1,14 @@
+<% content_for :message do %>
+  <p><%= t("mailer.general.greeting.formal", name: style_bold(@user.login)).html_safe %></p>
+
+  <p><%= t(".html.hidden", title: style_creation_link(@series.title, @series)).html_safe %></p>
+
+  <p><%= t(".access") %></p>
+
+  <p><%= t(".check_email") %></p>
+
+  <p><%= t(".html.tos_violation", tos_link: tos_link(t ".tos")).html_safe %></p>
```

Comment:
The ideal configuration of parentheses would be this:
```suggestion
  <p><%= t(".tos_violation.html", tos_link: tos_link(t(".tos"))) %></p>
```

Similar for the abuse link below.